### ISSUE TITLE
[Request] Patch release to unblock Ruby 2.6 compatibility

### ISSUE THREAD

**Issue by thePsguy** [realm/jazzy#1392]
I see that Ruby 2.6 support was fixed and submitted in https://github.com/realm/jazzy/issues/1388 after being broken in the 0.15.0 release, but there hasn't been a release since.

Reverting to an older version of Jazzy does not work due to the latest version requirement:
```
A new version of jazzy is available (your current local version is
    0.14.4 and the latest release version is 0.15.0).
Would you like to update jazzy to the latest release (REQUIRED)? [y/N] N
ERROR: The latest version of jazzy is required to generate
        Swift and Objective-C reference docs.
Exiting...
```

**Comment by johnfairh** [comment]
I don't think that error message is from any of our tools?

I'll put out a patch release -- I was leaving it to see if there actually were any other people stuck on old versions of Ruby.

**Comment by johnfairh** [comment]
Released 0.15.1
